webapp/src/store/getters.ts

```diff
@@ -0,0 +1,26 @@
+import { GetterTree } from 'vuex';
+import { AppState } from '../types';
+
+const getters: GetterTree<AppState, AppState> = {
+  totalAmountDonated: ({ transactions }) => {
+    return transactions.filter(t => t.type == 'donation' && t.amount > 0)
+      .map(t => t.amount)
+      .reduce((a, b) => a + b, 0);
+  },
+  totalAmountDistributed: ({ transactions, user }) => {
+    return transactions.filter(t => t.type === 'distribution' && user && t.to === user._id)
+      .map(t => t.amount)
+      .reduce((a, b) => a + b, 0);
+  },
```

Comment:
Only successful transactions should count, and distribution from this user will have `t.from` pointing to this user, not `t.to`

i.e. `t.type === 'distribution' && user && t.from === user._id && t.status === 'success'`